Packs/GraphQL/Integrations/GraphQL/GraphQL.py

```diff
@@ -70,7 +70,10 @@ def main() -> None:
 
         transport = RequestsHTTPTransport(**request_params)
         handle_proxy()
-        client = Client(transport=transport, fetch_schema_from_transport=True)
+        client = Client(
+            transport=transport,
+            fetch_schema_from_transport=params.get('fetch_schema_from_transport', True),
```

Comment:
you can get here None after upgrade as the server will put there a null after upgrade. See here: https://github.com/demisto/etc/issues/37415

Reply:
added handling for it